pkg/config/system_test.go

```diff
@@ -367,13 +367,14 @@ func TestComputeSplitKeyTableIDs(t *testing.T) {
 	baseSql, _ /* splits */ := schema.GetInitialValues()
 	// Real system tables plus some user stuff.
 	kvs, _ /* splits */ := schema.GetInitialValues()
+	start := uint32(schema.InitialUserDescriptorID())
```

Comment:
Laying the ground here for `start` to be outside of the reserved ID range. Again I don't want to use `keys` ID constants if I can avoid it.